module/utility/chat-dice.mjs

```diff
@@ -73,6 +73,11 @@ async function doRollAndGenerateChatMessage(chatMessageCmd) {
             roller.makeEntangleRoll();
             break;
 
+        case "t":
```

Comment:
question: Why do we need a separate type to roll transforms? They're just the pip value of the dice aren't they? This feels like overkill as success and adjustment are basically the same thing. Thoughts?

Reply:
Transform is just the pip value of the dice, correct. If we want to refactor Adjustment/Success/Transform and others that just total Pips into 1 roll type, I could get behind that. I wouldn't want to have Transform use the "makeAdjustmentRoll" code, however, just to avoid confusion.

Reply:
TODO: Remove Transform Roll

Reply:
Fixed. Transform now uses Effect rolls.